src/ApiNavigation.js

```diff
@@ -916,7 +918,7 @@ export class ApiNavigation extends AmfHelperMixin(LitElement) {
    */
   _createListMap(endpoints) {
     const map = {};
-    const getPathInit = (endpoint) => endpoint.path.split('/')[1];
+    const getPathInit = (endpoint) => this._getValue(endpoint, this.ns.raml.vocabularies.apiContract.path).split('/')[1];
```

Comment:
Is this a breaking change? I men, will it work with current model (2.0)?

Reply:
Yes, apiContract is part of current model. It wouldn't work with old model